experiments/AtmosLES/dycoms.jl

```diff
@@ -115,7 +141,8 @@ function integral_load_auxiliary_state!(
     aux::Vars,
 )
     FT = eltype(state)
-    integrand.radiation.attenuation_coeff = state.ρ * m.κ * aux.moisture.q_liq
+    integrand.radiation.attenuation_coeff =
+        state.ρ * m.κ * state.moisture.ρq_liq / state.ρ
```

Comment:
```suggestion
    integrand.radiation.attenuation_coeff =
        m.κ * state.moisture.ρq_liq
```

Reply:
I will prolly keep dycoms without rain and add rain to squall line case when it is merged. I just wanted to have something to compile for.